### DISCUSSION TITLE
1.5.3

### DISCUSSION THREAD

**Discussion by michael-s-molina** [apache/superset#22644] [General]
Hello Superset Community,

This is a call for the vote to release Apache Superset version 1.5.3.

The release candidate:
https://dist.apache.org/repos/dist/dev/superset/1.5.3rc1/

Git tag for the release:
https://github.com/apache/superset/tree/1.5.3rc1

The Change Log for the release:
https://github.com/apache/superset/blob/1.5.3rc1/CHANGELOG.md

The Updating instructions for the release:
https://github.com/apache/superset/blob/1.5.3rc1/UPDATING.md

public keys are available at:

https://www.apache.org/dist/superset/KEYS

The vote will be open for at least 72 hours or until the necessary number
of votes are reached.

Please vote accordingly:

[ ] +1 approve
[ ] +0 no opinion
[ ] -1 disapprove with the reason

Thanks,
The Apache Superset Team

<hr /><em>This discussion was created from the release <a href='https://github.com/apache/superset/releases/tag/1.5.3rc1'>1.5.3rc1</a>.</em>

**Comment by iercan** [comment]
Can you please include this pr [#21737 ](https://github.com/apache/superset/pull/21737) ?

**Reply by michael-s-molina** [reply]
Hi @iercan. 1.5.3 includes only security fixes. We highly recommend migrating to 2.0.1 to get the most up-to-date fixes. We'll probably drop support for 1.5 in the near future.

**Reply by iercan** [reply]
It is said that 1.5 is LTS. Is there any end of life date?

**Reply by michael-s-molina** [reply]
Yes, 1.5 is LTS, it was launched on Apr 25, 2022, and we're supporting it since then. The next release on the line will be 2.1 and at that point, the LTS version will probably be 2.0.x. We don't have a date yet but we are already recommending the migration. You can find more information about this in the [email we sent to the dev list](https://lists.apache.org/thread/hhtf07d8jsyjtl4shdzky500mpkfqry5).
